from_cpython/Include/Python.h

```diff
@@ -134,6 +134,8 @@ PyObject* PyGC_AddRoot(PyObject*) PYSTON_NOEXCEPT;
 //      PyGC_AddRoot(obj);
 // to reduce any chances of compiler reorderings or a GC somehow happening between the assignment
 // to the static slot and the call to PyGC_AddRoot.
+//
```

Comment:
Can we add a quick comment here saying that this is a pyston change and what it's for?